records.json

```
@@ -74,7 +75,7 @@
   "ns": {
     "easyassets": "ns1.vercel-dns.com",
```

Biome (2.4.15)

[error] The key easyassets was already declared.

(lint/suspicious/noDuplicateObjectKeys)